os2forms_failed_jobs.install

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Code related to installing and updating this module.
+ */
+
+/**
+ * Implements hook_schema().
+ *
+ * @phpstan-return array<string, mixed>
+ */
+function os2forms_failed_jobs_schema(): array {
+  $schema['os2forms_queue_submission_relation'] = [
```

Comment:
Isn't it customary to prefix the table name with the module name, i.e. call the table `os2forms_failed_jobs_submission_jobs`, say?

Reply:
Changed